### ISSUE TITLE
Simulcast sur les fiches animés

### ISSUE THREAD

**Issue by SubaruSumeragi** [mangaki/mangaki#48]
Dans un volonté de promouvoir la diffusion légale de l'animation japonaise, il serait intéressant que les fiches des animés comportent des liens vers les pages des séries des principaux sites de simulcasts disponibles en France (ADN, Waka, Crunchy, Viewster, Daisuki).

**Comment by jilljenn** [comment]
Oui, bonne idée. Ça implique de les contacter, leur proposer ces liens, leur demander leur catalogue, leur logo et éventuellement s'ils sont prêts à offrir des mois d'abonnement pour certains membres en échange.

**Comment by SubaruSumeragi** [comment]
Comme je le disais à Jill-Jenn vendredi au Kawaii, le but serait d'avoir la structure en place permettant cela avant de le proposer aux éditeurs. Ou, je viens d'y penser, le proposer aux éditeurs en annonçant un délai pour la mise en place (s'ils sont suffisamment intéresser pour cela).

**Comment by jilljenn** [comment]
Vous voyez juste un lien « Voir le premier épisode sur Wakanim » (+ logo) ou plus que ça ? Au-dessus de « Voir sur myAnimeList ».
Exemple : [Terror in Resonance](http://mangaki.fr/anime/215)

**Comment by SubaruSumeragi** [comment]
Je voyais un truc du même genre que "Références" : "Disponible sur : " suivi des liens + logo (car certaines séries sont disponibles sur plusieurs plateforme).
Mais lien + logo dans Référence ne me choque pas plus que cela vu qu'après tout, en soit, c'est un lien de référence ;)
